### PR TITLE
Update MTU's funding info

### PR DESCRIPTION
**Pull request by sgibson91**

### CODE REVIEW

File: config/clusters/2i2c/mtu.values.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ jupyterhub:
           name: 2i2c
           url: https://2i2c.org
         funded_by:
-          name: 2i2c
-          url: https://2i2c.org
+          name: ""
+          url: ""
```
Comment (sgibson91) [inline config/clusters/2i2c/mtu.values.yaml:27]: @lebrown do you have funding info for the hub please?

Comment (lebrown) [reply config/clusters/2i2c/mtu.values.yaml:27]: Michigan Technological University - College of Computing
https://www.mtu.edu/computing/

Comment (sgibson91) [reply config/clusters/2i2c/mtu.values.yaml]: Thanks!

**Comment by sgibson91** [comment]
I will self-merge this as it is only homepage info that is edited

**Pull request merged**